LocalMultiplayerAgent/Config/MultiplayerSettingsValidator.cs

```diff
@@ -33,17 +33,17 @@ public bool IsValid()
                 throw new Exception("OutputFolder or TitleId not found. Call SetDefaultsIfNotSpecified() before this method");
             }
 
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                Console.WriteLine("Running LocalMultiplayerAgent on Linux is not supported yet.");
-                return false;
-            }
-
-            if (Globals.GameServerEnvironment == GameServerEnvironment.Linux && !_settings.RunContainer)
-            {
-                Console.WriteLine("The specified settings are invalid. Using Linux Game Servers requires running in a container.");
-                return false;
-            }
+            // if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            // {
+            //     Console.WriteLine("Running LocalMultiplayerAgent on Linux is not supported yet.");
+            //     return false;
+            // }
+            //
+            // if (Globals.GameServerEnvironment == GameServerEnvironment.Linux && !_settings.RunContainer)
+            // {
+            //     Console.WriteLine("The specified settings are invalid. Using Linux Game Servers requires running in a container.");
+            //     return false;
+            // }
```

Comment:
Thanks for contributing! It seems that we should have some conditional logic in place as well to see if we are running in macos to prevent the check, rather than comment it out.  Similarly for the ProcessRunner start command. @dgkanatsios  thoughts?

Reply:
I'm happy to push another tidier commit. I wasn't sure if you'd be interested in MacOS support!

I can also add an example config